src/Broadway/EventStore/EventStoreInterface.php

```diff
@@ -28,6 +28,8 @@ public function load($id);
     /**
      * @param mixed                      $id
      * @param DomainEventStreamInterface $eventStream
+     *
+     * @throws EventStoreException
```

Comment:
I'm fine either way, as PHP doesn't actually have the concept of (un)checked exceptions, it might be nice to still document that we can throw both exceptions. But I do agree that it's an exception you probably can't recover from.